### ISSUE TITLE
[BUG] Cure Rot is broken

### ISSUE THREAD

**Issue by this-friday-night** [Rotwood-Vale/Ratwood-Keep#1471]
## Describe the bug
The Cure Rot spell doesn't work on rotten corpses.  When used, it only returns the text "Nothing happens."

## Round ID:
Unknown, but this is still happening after the recent fix attempt in #1416 

## Steps to Reproduce:
Play a divine caster (Acolyte or Cleric) w/Pestra selected.
Cast Cure Rot on a rotten corpse.
"Nothing happens." 

## Additional context
It still works on living zombies.
Surgeons can also de-rot corpses just fine.

**Comment by Nakalon** [comment]
What the fuck, this is the most clear-cut, cut and dry, open and shut, copy and paste, cock and balls case I ever did see!!

Fucking rot deadites and fartites!

**Comment by Gyrandola** [comment]
> ## Describe the bug
> The Cure Rot spell doesn't work on rotten corpses. When used, it only returns the text "Nothing happens."
> 
> ## Round ID:
> Unknown, but this is still happening after the recent fix attempt in #1416
> 
> ## Steps to Reproduce:
> Play a divine caster (Acolyte or Cleric) w/Pestra selected. Cast Cure Rot on a rotten corpse. "Nothing happens."
> 
> ## Additional context
> It still works on living zombies. Surgeons can also de-rot corpses just fine.

Right, i was there to see the bug live. What triggered it is the cure rot spell was used on someone who did NOT have the zombie antag datum. So the spell worked correctly. As for why the person did not have the zombie datum, that's another problem altogether.
